pkg/volume/csi/csi_mounter.go

```diff
@@ -328,9 +328,18 @@ func (c *csiMountMgr) podAttributes() (map[string]string, error) {
 }
 
 func (c *csiMountMgr) GetAttributes() volume.Attributes {
-	mounter := c.plugin.host.GetMounter(c.plugin.GetPluginName())
-	path := c.GetPath()
-	supportSelinux, err := mounter.GetSELinuxSupport(path)
+	kvh, ok := c.plugin.host.(volume.KubeletVolumeHost)
+	var (
+		supportSelinux bool
+		err            error
+	)
+	if !ok {
+		klog.V(2).Info(log("plugin volume host does not implement KubeletVolumeHost interface"))
```

Comment:
This seems like a fatal error if it fails.  Can we try to initialize host util earlier and pass it in through csiMountMgr?

Reply:
@msau42 I updated as you requested. the `csiMountMgr` struct is just created directly inside of `csi_mounter.go`, instead of a constructor of some type, so it's not quite as clean as I would have liked.  There are a few other places in `csi_mounter.go` where the same cast is done, and they don't treat it as fatal. If they had all treated it as fatal, I would have just done the cast once and saved it inside of the `csiPlugin` struct instead.

Reply:
I think checking in NewMounter() and NewUnmounter() are fine.